src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -420,10 +431,18 @@
 	margin-bottom: 0px;
 }
 
+.chat-warning .interactive-warning .codicon {
+	margin-top: 2px;
+}
+
 .interactive-response .interactive-response-error-details .codicon {
 	margin-top: 1px;
 }
 
+.chat-warning .interactive-warning .codicon-warning {
+	color: var(--vscode-notificationsWarningIcon-foreground) !important; /* Have to override default styles which apply to all lists */
```

Comment:
Does the contrast look ok in different themes? Do you end up with yellow on white in a light theme?

Reply:
Wouldn't the color token already have a light variant? It should use a darker yellow for light themes.

Reply:
I guess so, but text is thinner and I want to make sure we check

Reply:
![image](https://github.com/microsoft/vscode/assets/30305945/9b185849-089f-4fbc-973a-bc10316c27b1)

Reply:
LGTM

Reply:
Oops I read it as foreground color for the text